### DISCUSSION TITLE
No plans to compile to executables?

### DISCUSSION THREAD

**Discussion by lf-araujo** [fubark/cyber#98] [General]
The language is designed as a scripting language, but are there any plans in allowing compilation to executable binaries?

Thanks and congrats for this project.

**Comment by fubark** [comment]
Yes, this is something we want: https://github.com/fubark/cyber/issues/46

I haven't announced this yet but Cyber is also a compiled language, but the immediate plan for that is to transpile it to C++.
However, you'll also be able to bundle the source or bytecode of your program into an exe. The drawback there is it incurs a minimum size for the interpreter but I don't think it will be significant.
